service-binding-cosmosdb-sql/README.md

```diff
@@ -4,7 +4,7 @@ This sample shows how to run app with Azure Cosmos DB(SQL) in `Azure Spring Clou
 
 ### Prerequisite
 
-* [JDK 8](https://docs.microsoft.com/en-us/azure/java/jdk/java-jdk-install)
+* [JDK 11](https://docs.microsoft.com/en-us/azure/java/jdk/java-jdk-install)(Doesn't work on Java8)
```

Comment:
Do we know why?

Reply:
It's a bug of azure-spring-boot.

Reply:
Have we created an issue about it?
Sounds like we should fix it ASAP.

Reply:
Already have a discussion with Jialin. There will be release a fix for it.